client/src/views/Navbar.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+    <md-toolbar class="md-primary" md-elevation="1">
+        <md-avatar>
+          <img src="../assets/thm_logo.png">
+        </md-avatar>
+        <h3 class="md-title" style="flex: 1">The Halo Mod</h3>
+          <router-link to="/"><md-button class="md-primary">Home</md-button></router-link>
+          <router-link to="/about"><md-button class="md-primary">About</md-button></router-link>
+      </md-toolbar>
+</template>
+
+<script>
+// @ is an alias to /src
+
+export default {
+  name: 'Navbar',
+  components: {
+  },
+};
+</script>
```

Comment:
Should this be in a `components` folder instead of `views`? It seems like views is for routes. Not a big deal either way. Just might help with organization as we move along.

Reply:
Yes, you're right! I forgot there really should be a "Components" folder. Fixing that :)